app/src/pages/admin/pluginsPage/pluginsTabs/installedTab/installedTab.jsx

```diff
@@ -1,23 +1,169 @@
+import { IntegrationBreadcrumbs } from 'pages/common/settingsPage/integrationsTab/integrationBreadcrumbs';
+import {
+  IntegrationInfoContainer,
+  IntegrationSettingsContainer,
+} from 'components/integrations/containers';
+import { INTEGRATION_NAMES_TITLES } from 'components/integrations';
 import { InputDropdown } from 'components/inputs/inputDropdown';
+import {
+  INSTALLED_PLUGINS_SUBPAGE,
+  INSTALLED_PLUGINS_SETTINGS_SUBPAGE,
+  SUB_PAGES_SEQUENCE,
+  DEFAULT_BREADCRUMB,
+} from './constants';
 import styles from './installedTab.scss';
-import { PluginsFilter } from './../../pluginsFilter';
-import { PluginsListItems } from './../../pluginsListItems';
+import { PluginsFilter } from '../../pluginsFilter';
+import { PluginsListItems } from '../../pluginsListItems';
 
 const cx = classNames.bind(styles);
 
+const messages = defineMessages({
+  disabledPluginMessage: {
+    id: 'PluginItem.disabledPluginMessage',
+    defaultMessage: 'Plugin has been disabled',
+  },
+  enabledPluginMessage: {
+    id: 'PluginItem.enabledPluginMessage',
+    defaultMessage: 'Plugin has been enabled',
+  },
+});
+
+@injectIntl
+@connect(null, {
+  showNotification,
+  updatePluginLocallyAction,
+})
 export class InstalledTab extends Component {
   static propTypes = {
+    intl: intlShape.isRequired,
     filterItems: PropTypes.array.isRequired,
     plugins: PropTypes.array.isRequired,
+    updatePluginLocallyAction: PropTypes.func.isRequired,
+    showNotification: PropTypes.func,
+  };
+
+  static defaultProps = {
+    showNotification: () => {},
   };
 
   state = {
     activeFilterItem: ALL_GROUP_TYPE,
+    subPage: DEFAULT_BREADCRUMB,
+  };
+
+  onToggleActive = (itemData) => {
+    const {
+      intl: { formatMessage },
+    } = this.props;
+    const toggleActive = !itemData.enabled;
+
+    return fetch(URLS.pluginUpdate(itemData.type), {
+      method: 'PUT',
+      data: {
+        enabled: toggleActive,
+      },
+    }).then(() => {
+      const plugin = {
+        ...itemData,
+        enabled: toggleActive,
+      };
+
+      this.props.updatePluginLocallyAction(plugin);
+      this.props.showNotification({
+        type: NOTIFICATION_TYPES.SUCCESS,
+        message: toggleActive
+          ? formatMessage(messages.enabledPluginMessage)
+          : formatMessage(messages.disabledPluginMessage),
+      });
+    });
+  };
+
+  getPageContent = () => {
+    const {
+      subPage: { type, data, title },
+      activeFilterItem,
+    } = this.state;
+    const { filterItems } = this.props;
+    const newData = this.props.plugins.find((item) => item.type === data.type);
+
+    switch (type) {
+      case INSTALLED_PLUGINS_SUBPAGE:
+        return (
+          <IntegrationInfoContainer
+            integrationType={newData}
+            isGlobal
+            onToggleActive={(itemData) => this.onToggleActive(itemData)}
+            onItemClick={(pageData, pageTitle) =>
```

Comment:
This handler can be extracted to a class method. This will prevent unnecessary renders